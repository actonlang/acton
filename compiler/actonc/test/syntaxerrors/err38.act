# Hex literal error
def bad_hex():
    return 0xGHI