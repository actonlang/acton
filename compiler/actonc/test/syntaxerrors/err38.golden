Building file test/syntaxerrors/err38.act using temporary scratch directory
[error Parse error]: unexpected 'G'
                     expecting hexadecimal integer

     +--> test/syntaxerrors/err38.act@3:14-3:15
     |
   3 |     return 0xGHI
     :              ^ 
     :              `- unexpected 'G'
     :                 expecting hexadecimal integer
     :                 
-----+
